Maddori.Apple/Maddori.Apple/Screen/MyBox/UIComponent/MyFeedbackCollectionView.swift

```diff
@@ -0,0 +1,114 @@
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewFlowLayout)
+        collectionView.backgroundColor = .backgroundWhite
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(MyFeedbackHeaderView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: MyFeedbackHeaderView.className)
+        collectionView.register(MyFeedbackCollectionViewCell.self, forCellWithReuseIdentifier: MyFeedbackCollectionViewCell.className)
+        return collectionView
+    }()
+    
+    // MARK: - life cycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) { nil }
+    
+    private func render() {
+        self.addSubview(feedbackCollectionView)
+        feedbackCollectionView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+    }
+}
+
+    // MARK: - extension
+
+extension MyFeedbackCollectionView: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+        guard let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: MyFeedbackHeaderView.className, for: indexPath) as? MyFeedbackHeaderView else { return UICollectionReusableView() }
+        header.setCssLabelText(with: indexPath.section)
+        switch kind {
+        case UICollectionView.elementKindSectionHeader:
+            return header
+        default:
+            return UICollectionReusableView()
+        }
+    }
+}
+extension MyFeedbackCollectionView: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        mockData.filter { $0.type == FeedBackType.allCases[section] }.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: MyFeedbackCollectionViewCell.className, for: indexPath) as? MyFeedbackCollectionViewCell else { return UICollectionViewCell()}
+        switch indexPath.section {
+        case 0:
+            let data = mockData.filter { $0.type == .continueType }
+            cell.setCellLabel(title: data[indexPath.item].title, content: data[indexPath.item].content)
+        case 1:
+            let data = mockData.filter { $0.type == .stopType }
+            cell.setCellLabel(title: data[indexPath.item].title, content: data[indexPath.item].content)
+        default:
+            let data = mockData.filter { $0.type == .startType }
+            cell.setCellLabel(title: data[indexPath.item].title, content: data[indexPath.item].content)
+        }
+        return cell
+    }
```

Comment:
p4;

```suggestion
    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: MyFeedbackCollectionViewCell.className, for: indexPath) as? MyFeedbackCollectionViewCell else { return UICollectionViewCell()}
        var data: [FeedBack] = []
        switch indexPath.section {
        case 0:
            data = mockData.filter { $0.type == .continueType }
        case 1:
            data = mockData.filter { $0.type == .stopType }
        default:
            data = mockData.filter { $0.type == .startType }
        }
        cell.setCellLabel(title: data[indexPath.item].title, content: data[indexPath.item].content)
        return cell
    }
```
여기 `setCellLabel`이 모든 케이스마다 나와야 할까 생각하다가 안그래도 될 방법을 찾아서 suggestion 남겨봤습니다!
뭔가 코드 수가 많이 줄어들 줄 알았는데 한 줄 줄어드네요 ㅎㅎ;;

Reply:
오호 ! 안그래도 같은 함수를 똑같이 세번써서 찝찝한 상태로 넘어갔는데.... 깔끔하게 고칠 수 있군요 !
찝찝해도 넘어가는 버릇을 고쳐야겠습니다 !
감사합니다 이드 ❤️

Reply:
오 가독성이 훨씬 좋아요!